editoast/editoast_models/src/app_health.rs

```diff
@@ -0,0 +1,13 @@
+use diesel::sql_query;
+use diesel_async::RunQueryDsl;
+
+use crate::DbConnection;
+use crate::EditoastModelsError;
+
+pub struct AppHealth;
+
+impl AppHealth {
+    pub async fn run_simple_request(conn: &mut DbConnection) -> Result<usize, EditoastModelsError> {
+        Ok(sql_query("SELECT 1").execute(conn).await?)
```

Comment:
Ideally, we should set a timeout for this query. Let's say the database takes more than `x` ms to respond the health should respond with a 500. Same for redis.